tests/pytests/integration/modules/test_mac_system.py

```diff
@@ -0,0 +1,298 @@
+"""
+integration tests for mac_system
+"""
+
+import logging
+
+import pytest
+from saltfactories.utils import random_string
+
+log = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.flaky(max_runs=10),
+    pytest.mark.slow_test,
+    pytest.mark.skip_if_not_root,
+    pytest.mark.skip_unless_on_darwin,
+    pytest.mark.usefixtures("salt_sub_minion"),
+    pytest.mark.skip_if_binaries_missing("systemsetup"),
+    pytest.mark.skip_initial_gh_actions_failure,
+]
+
+
+@pytest.fixture(scope="function")
+def setup_teardown_vars(salt_call_cli):
+    ret = salt_call_cli.run("service.enabled", "com.apple.atrun")
+    ATRUN_ENABLED = ret.data
+
+    ret = salt_call_cli.run("system.get_remote_login")
+    REMOTE_LOGIN_ENABLED = ret.data
+
+    ret = salt_call_cli.run("system.get_remote_events")
+    REMOTE_EVENTS_ENABLED = ret.data
+
+    ret = salt_call_cli.run("system.get_subnet_name")
+    SUBNET_NAME = ret.data
+
+    ret = salt_call_cli.run("system.get_disable_keyboard_on_lock")
+    KEYBOARD_DISABLED = ret.data
+
+    try:
+        yield
+    finally:
+        if not ATRUN_ENABLED:
+            atrun = "/System/Library/LaunchDaemons/com.apple.atrun.plist"
+            salt_call_cli.run("service.stop", atrun)
+
+        salt_call_cli.run("system.set_remote_login", REMOTE_LOGIN_ENABLED)
+        salt_call_cli.run("system.set_remote_events", REMOTE_EVENTS_ENABLED)
+        salt_call_cli.run("system.set_subnet_name", SUBNET_NAME)
+        salt_call_cli.run("system.set_disable_keyboard_on_lock", KEYBOARD_DISABLED)
```

Comment:
```suggestion
@pytest.fixture(scope="function", autouse=True)
def _setup_teardown_vars(salt_call_cli):
    ret = salt_call_cli.run("service.enabled", "com.apple.atrun")
    ATRUN_ENABLED = ret.data

    ret = salt_call_cli.run("system.get_remote_login")
    REMOTE_LOGIN_ENABLED = ret.data

    ret = salt_call_cli.run("system.get_remote_events")
    REMOTE_EVENTS_ENABLED = ret.data

    ret = salt_call_cli.run("system.get_subnet_name")
    SUBNET_NAME = ret.data

    ret = salt_call_cli.run("system.get_disable_keyboard_on_lock")
    KEYBOARD_DISABLED = ret.data

    try:
        yield
    finally:
        if not ATRUN_ENABLED:
            atrun = "/System/Library/LaunchDaemons/com.apple.atrun.plist"
            salt_call_cli.run("service.stop", atrun)

        salt_call_cli.run("system.set_remote_login", REMOTE_LOGIN_ENABLED)
        salt_call_cli.run("system.set_remote_events", REMOTE_EVENTS_ENABLED)
        salt_call_cli.run("system.set_subnet_name", SUBNET_NAME)
        salt_call_cli.run("system.set_disable_keyboard_on_lock", KEYBOARD_DISABLED)
```

?

Since it's not getting used anywhere....